pom.xml

```diff
@@ -24,6 +25,12 @@
       <version>3.8.1</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter</artifactId>
+      <version>RELEASE</version>
```

Comment:
It looks like this version should be replaced with ${junit.jupiter.version} from the properties section above